orderer/common/channelparticipation/responses.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package channelparticipation
+
+type ErrorResponse struct {
+	Error string `json:"error"`
+}
+
+// ListAllChannels carries the response to a List of all channels.
+type ListAllChannels struct {
+	Size          int                `json:"size"`          // The number of channels
```

Comment:
Why do you need this? Isn't it just length of the `Channels` slice?

Reply:
It is, but it goes to the client via JSON, so maybe it is useful there?

Reply:
See comment to @yacovm below...

Reply:
Out it goes.